src/Symfony/Component/DependencyInjection/Tests/Compiler/ResolveReferencesToAliasesPassTest.php

```diff
@@ -123,6 +123,28 @@ public function testDeprecationNoticeWhenReferencedByDefinition()
         $this->process($container);
     }
 
+    public function testNoDeprecationNoticeWhenReferencedByDeprecatedAlias()
```

Comment:
Tests don't cover the case of a deprecated alias referenced by a deprecated definition. You should be adding another test

Reply:
done